diesel/src/macros/mod.rs

```diff
@@ -810,6 +814,9 @@ macro_rules! __diesel_table_impl {
                 }
             }
 
+            // impl $crate::expression::AppearsOnTable<$crate::query_builder::Only<table>> for <table as $crate::query_source::QuerySource>::DefaultSelection {}
```

Comment:
This resulted in an impl conflict error, but apparently was not needed after all.

Reply:
Yes this is not needed if we implement the corresponding traits for columns, as this is then implied by a more general impl.